library/lib/core/application.cpp

```diff
@@ -149,6 +150,7 @@ void Application::createWindow(std::string windowTitle)
 bool Application::mainLoop()
 {
     static ControllerState oldControllerState = {};
+    static View* firstResponder;
```

Comment:
`touchFirstResponder` ?

Reply:
I think it will be used also for mouse , so may be leave it as is?